linux_files/setup

```diff
@@ -89,44 +76,89 @@ function byemessage {
     exit 0
 }
 
+function mainmenu{
+
+SELECTION=$(whiptail --title "WLinux Setup Main Menu" --menu "Enter Selects" 0 0 0 --cancel-button Quit --ok-button Select \
```

Comment:
find the solution.
from line 79 to line 81, it should be:
```
function mainmenu {
SELECTION=$(...
```
it seems weird that bash have such behaviour...

Reply:
So, there can't be a line? Weird. That's why it was working when I copied and pasted but not run.

Reply:
Yeah, me too, bash has weird bahvior